hubspot_commerce.module

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Hubspot_commerce.module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+const HUBSPOT_COMMERCE_LOGGER_CHANNEL = 'hubspot_commerce';
```

Comment:
**CHANGE REQUEST**

Rename to `commerce_hubspot` following the renaming of the module.